src/Models/ContentContentHighlightPropsHighlightSettings.php

```diff
@@ -0,0 +1,72 @@
+<?php declare(strict_types=1);
+
+namespace Relewise\Models;
+
+/** Encapsulates how search highlighting is to work. */
+class ContentContentHighlightPropsHighlightSettings
```

Comment:
`ContentContent` er det meningen?

Reply:
Yes, it is 2 generic parameters. The first `Content` and second `ContentHighlightProps` for the generic type `HighlightSettings`. But I agree that the names become very long and weird when we expose generic types in the API.